configure.ac

```diff
@@ -177,6 +177,24 @@ AC_ARG_ENABLE([zmq],
   [use_zmq=$enableval],
   [use_zmq=yes])
 
+AC_ARG_ENABLE([ssse3],
+  [AC_HELP_STRING([--enable-ssse3],
```

Comment:
`AC_HELP_STRING` is obsolete with the current tool set:
[autoconf-2.69 Obsolete-Macros](https://www.gnu.org/software/autoconf/manual/autoconf-2.69/html_node/Obsolete-Macros.html)
I think we should use `AS_HELP_STRING` instead.